### DISCUSSION TITLE
Error raised during explode of blocks

### DISCUSSION THREAD

**Discussion by mag06998** [mozman/ezdxf#1044] [Issues]
I'm writing a program right now to get the sectional properties of various extrusions, which are contained in individual .dxf files. One of the things I'd like to do to preprocess the .dxf files is by exploding all blocks. Here's the basic code:

    for file in files:
        dxf = ezdxf.readfile(file)
        msp = dxf.modelspace()
        
        blocks = msp.query("INSERT")
        if blocks.first != None:
            for ent in blocks:
                ent.explode()
        dxf.saveas(file)

However, I'm getting this error when attempting to do so:

copy process ignored ACDB_TEXTOBJECTCONTEXTDATA_CLASS(#BE50) - this may cause problems in AutoCAD
copy process ignored ACDB_TEXTOBJECTCONTEXTDATA_CLASS(#BF3F) - this may cause problems in AutoCAD
copy process ignored ACDB_TEXTOBJECTCONTEXTDATA_CLASS(#BFB8) - this may cause problems in AutoCAD
copy process ignored ACDB_TEXTOBJECTCONTEXTDATA_CLASS(#BFBF) - this may cause problems in AutoCAD
copy process ignored ACDB_TEXTOBJECTCONTEXTDATA_CLASS(#BFB8) - this may cause problems in AutoCAD
copy process ignored ACDB_TEXTOBJECTCONTEXTDATA_CLASS(#BFBF) - this may cause problems in AutoCAD

When I open up the .dxf in AutoCAD, everything appears correct and everything has been exploded, but when I create geometries in the sectionproperties library I am using unexpected things are happening with the geometry (polylines having been shifted to locations where they shouldn't be, for example). I believe it's related what I'm doing with ezdxf because I haven't had any issues when modifying the files manually. What does this error seem to indicate? Any suggestions for another way to go about this?

**Comment by mozman** [comment]
I don't really know how to answer this.

1. The warning states that problems may occur in AutoCAD because the DXF file contains unknown/undocumented elements that cannot be copied.
2. Problems occur in AutoCAD.
3. Answer: There may be problems in AutoCAD because necessary data may not have been copied.

You can try load the DXF file with the `RECOVER` command, but AutoCAD usually doesn't give usable error messages.
The typical behavior of the market leader, therefore my recommendation: if possible, never use the market leader's products: Autodesk, Adobe, Microsoft, ... The competition is much more eager to help and support the customer.
